pkg/support/dump.go

```diff
@@ -182,11 +187,14 @@ func (d *managerDumper) DumpZookeeperLog(basedir string) error {
 func (d *managerDumper) copyFromPod(namespace, pod, srcDir, dstDir string) error {
 	ioStreams, _, out, errOut := genericclioptions.NewTestIOStreams()
 	copyOptions := cp.NewCopyOptions(ioStreams)
-	copyOptions.Clientset = d.clientSet
-	copyOptions.ClientConfig = d.restConfig
-	copyOptions.Namespace = namespace
-
-	if err := copyOptions.Run([]string{pod + ":" + srcDir, dstDir}); err != nil {
+	factory := util.NewFactory(newRestClientGetter(namespace, d.restConfig))
+	if err := copyOptions.Complete(factory, cp.NewCmdCp(factory, ioStreams), []string{pod + ":" + srcDir, dstDir}); err != nil {
+		return fmt.Errorf("failed to complete: %v", err)
```

Comment:
nit: better to add some context about copy operation in this log, like `return fmt.Errorf("could not run copy operation: %v", err)` in ilne 198

Reply:
Fixed. Thanks!